feat(Python): Table of sensitive data collected by Python FastAPI integration (#6157)
getsentry/sentry-docs: src/platforms/python/guides/fastapi/index.mdx

</Note>

> ## Data collected

Reviewer: ```suggestion
## Data Collected
```

Author: changed capitalization of the headline in the last commit

> Here an overview of all the data that is collected by the FastAPI integration

Reviewer: ```suggestion
Here's an overview of all the data that's collected by the FastAPI integration:
```

Author: not there anymore when we have two tables.

> |                                                                       |                                **DEFAULT**                                |                                                                      |                                                                                |                                                                                |

Reviewer: Is it possible to add a heading to each of the 5 columns? It looks strange to just have one. Also, is it possible to create a table that has lines in it? It would make for a much faster/easier read for the users.

<img width="772" alt="Screenshot 2023-01-27 at 9 21 26 AM" src="https://user-images.githubusercontent.com/26800704/215152706-0ba8df3a-0a91-42a1-9e28-f0a0e81d95e5.png">

Author: Hey! Thanks for the review.
Unfortunately the tables are very limited. But I have talked to @Jesse-Box (designer) and he suggested to split the table in two, to make the information more accessible. I am trying this right now.

|                                                                       | **Performance:<br/>Off<sup>1)</sup> <br/>Send PII:<br/>Off<sup>3)</sup>** | Performance:<br/>On<sup>2)</sup> <br/>Send PII:<br/>Off<sup>3)</sup> |      Performance:<br/>Off<sup>1)</sup> <br/>Send PII:<br/>On<sup>4)</sup>      |      Performance:<br/>On<sup>2)</sup> <br/>Send PII:<br/>On<sup>4)</sup>       |
| Data sent when an <br/> unhandled error occurs                        |               <span style="color: #C83852;">**Yes**</span>                |               <span style="color: #C83852;">Yes</span>               |                    <span style="color: #C83852;">Yes</span>                    |                    <span style="color: #C83852;">Yes</span>                    |
| Data sent on <br/> each request                                       |                <span style="color: #69C289;">**No**</span>                |               <span style="color: #C83852;">Yes</span>               |                    <span style="color: #69C289;">No</span>                     |                    <span style="color: #C83852;">Yes</span>                    |

> | HTTP Headers                                                          |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>5)</sup></span> | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>5)</sup></span> |

Reviewer: ```suggestion
| HTTP headers                                                          |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>5)</sup></span> | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>5)</sup></span> |
```

Author: included in the last commit

| Cookies                                                               |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>6)</sup></span> | <span style="color: #D79528;">Sent <br/> Denylist applied <sup>5)</sup></span> |

> | Request Body                                                          |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |

Reviewer: ```suggestion
| Request body                                                          |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |
```

Author: included in last commit

> | Information about logged in User                                      |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |

Reviewer: ```suggestion
| Information about logged-in user                                      |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |
```

> | Users IP Address                                                      |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |

Reviewer: ```suggestion
| Users' IP address                                                      |             <span style="color: #69C289;">**Not Sent**</span>             |            <span style="color: #69C289;">Not Sent</span>             |                   <span style="color: #C83852;">Sent</span>                    |                   <span style="color: #C83852;">Sent</span>                    |
```

Author: included in the last commit
